services/src/ogc/wms/request.rs

```diff
@@ -29,29 +29,45 @@ pub enum GetCapabilitiesFormat {
     TextXml, // TODO: remaining formats
 }
 
+// TODO: remove serde aliases and use serde-aux and case insensitive keys
 #[derive(PartialEq, Debug, Deserialize, Serialize)]
 pub struct GetMap {
+    #[serde(alias = "VERSION")]
```

Comment:
couldn't we use serde_aux here?

Reply:
https://docs.rs/serde-aux/0.6.1/serde_aux/container_attributes/fn.deserialize_struct_case_insensitive.html

I think it won't work because it would need to be within the enum variant.